docs/index.html

```diff
@@ -46,25 +46,25 @@
 	<div class="content">
 
 <div class="container p-0">
-	<img class="w-100" src="&#x2F;images&#x2F;htmx_2025.png" alt="PHPScript&#x20;&#x28;HTMX&#x29;&#x3A;&#x20;Postmodern&#x20;front&#x20;end&#x20;interactions&#x20;in&#x20;PHP">
+	<img class="w-100" src="&#x2F;images&#x2F;homelab_2025.png" alt="Hosting&#x20;a&#x20;PHP&#x20;app&#x20;in&#x20;a&#x20;Homelab">
 </div>
 <div class="container bg-white pt-5">
 	<div class="row px-3 pb-5">
 		<div class="col-md-12">
 			<p>Join us on
-				<time datetime="2025-05-08T19:00:00-04:00" style="display:inline">May 8th, 2025</time>
-				as Jose Diaz-Gonzalez presents</p>
-			<h2 class="mb-4 font-weight-bold">PHPScript (HTMX): Postmodern front end interactions in PHP</h2>
+				<time datetime="2025-06-12T19:00:00-04:00" style="display:inline">June 12th, 2025</time>
+				as Daniel Abernathy presents</p>
+			<h2 class="mb-4 font-weight-bold">Hosting a PHP app in a Homelab</h2>
 			<p class="m-0">
-				<p>Interactivity in applications has always been important to providing a world-class user experience, but modern
-frontend frameworks can make this difficult for small teams or those without frontend experience. We've been
-writing interactive applications for decades, so has  anything changed for PHP developers? This talk will
-cover the history of how PHP developers have typically built applications, and how building great experiences
-may not always need new languages and frameworks.</p>
+				<p>We all love fun and exotic ways to deploy web applications, but sometimes you just want to keep things simple.
+You might reach for a VPS - but why not try hosting that app from your house? With Cloudflare Tunnels and a
+cheap mini PC you can easily and securely deploy a web application directly from your closet!</p>
 
 			</p>
-			<h3>Presented by Jose Diaz-Gonzalez</h3>
-			<p><p>CakePHP Core Developer and Platform Engineer</p>
+			<h3>Presented by Daniel Abernathy</h3>
+			<p><p>Daniel Abernathy has been a web developer for 13 years.
```

Comment:
The nested `<p>` doesn't look right.  Nothing you did, but something that should be looked at outside this PR